### PR TITLE
[docs] migrate simple-gateway to v1alpha2

### PR DESCRIPTION
**Pull request by ccfishk**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

document simple-gateway v1alpha2 

**Which issue(s) this PR fixes**:

Fixes #
translate simple-gateway from v1alpha1 to v1alpha2 

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: site-src/v1alpha2/guides/simple-gateway.md [hunk 1]
```diff
@@ -21,12 +18,10 @@ HTTP traffic on port 80. This particular GatewayClass automatically assigns an
...
 
-Gateways bind Routes to themselves via label selection (similar to how Services
-label select across Pod labels). In this example, the `prod-web` Gateway will
-bind any HTTPRoute resources which have the `gateway: prod-web-gw` label. The
-label can be any arbitrary label, but using one that identifies the name or
-capabilities of the Gateway is useful to Route owners and makes the relationship
-more explicit. More complex bi-directional matching and permissions are possible
+An `HTTPRoute` resource must be configured with `ParentRefs` which reference the parent
+[Gateway]gateway(s) that it should be attached to. Through this configuration, it can receive
+traffic from the gateway. And the BackendRefs defines the backend service where the traffic
```
Comment (bowei) [inline site-src/v1alpha2/guides/simple-gateway.md:23]: Gateway

**Comment by bowei** [comment]
Minor suggestion -- otherwise everything looks good.

File: site-src/v1alpha2/guides/simple-gateway.md (hunk 1 above)
Comment (ccfishk) [reply site-src/v1alpha2/guides/simple-gateway.md]: take the suggestion.

**Comment by ccfishk** [comment]
Found the pr is based on the forked repo which has different data from root repo. 
Suspicious the un-usual test failure is due to that reason.
Close this PR and migrate to a new PR soon.

**Pull request closed without merging**